libdnf-plugins/actions/actions.cpp

```diff
@@ -0,0 +1,725 @@
+                               "POST_TRANS hooks"),
+                            path.native(),
+                            line_number);
+                    }
+                }
+
+                auto direction = line.substr(direction_pos, command_pos - direction_pos - 1);
+                if (pkg_filter.empty() && !direction.empty()) {
+                    throw ActionsPluginError(
+                        M_("Error in file \"{}\" at line {}: Cannot use direction without package filter"),
+                        path.native(),
+                        line_number);
+                }
+
+                Action act;
+                act.file_path = path;
+                act.line_number = line_number;
+                act.pkg_filter = pkg_filter;
+                if (direction == "in") {
+                    act.direction = Action::Direction::IN;
+                } else if (direction == "out") {
+                    act.direction = Action::Direction::OUT;
+                } else if (direction == "") {
+                    act.direction = Action::Direction::ALL;
+                } else {
+                    throw ActionsPluginError(
+                        M_("Error in file \"{}\" at line {}: Unknown package direction \"{}\""),
+                        path.native(),
+                        line_number,
+                        direction);
+                }
+
+                act.args = split(line.substr(command_pos));
+                if (act.args.empty()) {
+                    throw ActionsPluginError(
+                        M_("Error in file \"{}\" at line {}: Missing command"), path.native(), line_number);
+                }
+                act.command = act.args[0];
+
+                switch (hook) {
+                    case Hooks::PRE_BASE_SETUP:
+                        pre_base_setup_actions.emplace_back(std::move(act));
+                        break;
+                    case Hooks::POST_BASE_SETUP:
+                        post_base_setup_actions.emplace_back(std::move(act));
+                        break;
+                    case Hooks::PRE_TRANS:
+                        pre_trans_actions.emplace_back(std::move(act));
+                        break;
+                    case Hooks::POST_TRANS:
+                        post_trans_actions.emplace_back(std::move(act));
+                }
+            }
+        }
+    }
+}
+
+void Actions::process_command_output_line(std::string_view line) {
+    auto eq_pos = line.find('=');
+    if (line.starts_with("tmp.")) {
+        std::string var_name(line.substr(4, eq_pos - 4));
+        if (eq_pos == std::string::npos) {
+            tmp_variables.erase(var_name);
+        } else {
+            tmp_variables[var_name] = line.substr(eq_pos + 1);
+        }
+        return;
+    }
+    if (eq_pos == std::string::npos) {
+        base.get_logger()->error(
+            "Actions plugin: Syntax error: Command returns line without eq character \"{}\"", std::string(line));
```

Comment:
How about saying `"... Missing equal sign (=) in command output line \"{}\""`?

Reply:
OK